.circleci/config.yml

```diff
@@ -0,0 +1,120 @@
+version: 2
+
+defaults: &defaults
+  working_directory: ~/project/yarn
+  docker:
+    - image: yarnpkg/dev:latest
+
+default_filters: &default_filters
+  tags:
+    only: '/^v[0-9]\+\.[0-9]\+\.[0-9]\+$/'
+
+jobs:
+  install:
+    <<: *defaults
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v1-yarn-{{ .Branch }}-{{ checksum "yarn.lock" }}
+            - v1-yarn-{{ .Branch }}-
+            - v1-yarn-
+      - run:
+          name: Install Dependencies
+          command: yarn install
+      - save_cache:
+          key: v1-yarn-{{ .Branch }}-{{ checksum "yarn.lock" }}
+          paths:
+            - .cache/yarn
+      - save_cache:
+          key: v1-yarn-{{ .Branch }}-
+          paths:
+            - .cache/yarn
```

Comment:
you don’t need to save multiple times!

Always save `v1-yarn-{{ .Branch }}-{{ checksum "yarn.lock" }}` and in order the 3:
- `v1-yarn-{{ .Branch }}-{{ checksum "yarn.lock" }}`
- `v1-yarn-{{ .Branch }}-`
- `v1-yarn-`

The `restore_cache` command basically works as `<key>.*`, so when you load `v1-yarn-{{ .Branch }}-` it will restore the last cache that matches `v1-yarn-{{ .Branch }}-.*`

Reply:
Thank you for your precious advice 🙇 
Fixed. c5e4c5a